demo/rails/app/views/captchas/index.html.erb

```diff
@@ -35,7 +35,7 @@
     <div id="example3"></div>
     <%= submit_tag %>
   <% end %>
-  <script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" async defer></script>
+  <script src="https://www.recaptcha.net/recaptcha/api.js?onload=onloadCallback&render=explicit" async defer></script>
```

Comment:
```suggestion
  <script src="<%= Recaptcha.configuration.api_server_url %>?onload=onloadCallback&render=explicit" async defer></script>
```

Reply:
that would work great and makes a lot more sense.